doc/todo/RELEASE

```diff
@@ -25,238 +13,66 @@ Following shell expansions are used in this document:
+  - [ ] Check if the debian package build log was successfull (debian/<DEBIAN_CODENAME>/elektra_*.buildinfo)
+- Check if all artifacts are present
+  - [ ] Check debian/<DEBIAN_CODENAME> if all debian packages were built
+  - [ ] Unarchive the elektra tarball and briefly verify if the correct version was archived
+- Inspect the changes to the libelektra git repository (*.bundle)
+  Inspect the git log of the master and debian branch with: git log -p --author="jenkins-release"
+  Master:
+  - [ ] Check diff of commit "Regenerate dot of plugins for release <VERSION>" and if 
+        doc/images/plugins.pdf was generated correctly (Only if plugins were modified)
+  - [ ] Check diff of commit "Update contract for release <VERSION>": Plugin tags in the plugin 
+        README.md should be sorted (Only if tags are not sorted)
+  Debian:
+  - [ ] Check if master is merged to debian branch (commit: "merge <VERSION>")
+  - [ ] Check diff of commit "Debian Package <VERSION>-1 (UNRELEASED)":
+        An entry for a new upstream version should have been created in debian/changelog
+  - [ ] Check diff of commit "Debian Package <VERSION>-1":
+        The previously created version entry in debian/changelog should have been updated from
+        UNRELEASED to unstable. 
+- Inspect the changes to the docu git repository (docu.bundle)
+  - [ ] Check diff of the latest commit "<VERSION> release" with: git log -p 
+- Inspect the website from the pipeline
+
+== Preperation ==
+
+- Finalize release notes without hash sums (move to final name, add links where to download release)
+- cp doc/todo/NEWS.md doc/news/_preparation_next_release.md
 
 = Publish =
 
-== Debian Package ==
-
- cd ~elektra/$VERSION/debian/$DVERSION && reprepro --ignore=wrongdistribution include stretch elektra_${DVERSION}_`dpkg-architecture -qDEB_BUILD_ARCH`.changes && update-reprepro
-
-== Commit source package ==
+- Trigger release pipeline without dry run
+- Add hash sums to release notes and run linkchecker to verify that all download links are working:
+  mkdir build && cd build && make html && ../scripts/link-checker external-links.txt
+- (Possible) rebuilds from website for last changes in the release notes
 
-Commit elektra-ftp:
- cd ~elektra-ftp && git add releases/elektra-$VERSION.tar.gz* && git commit -a -m "Release $VERSION"
-
-
-== push ==
-
-Now we know that everything worked well:
-
-Now push release branch with tags to github:
- cd ~e && git push && git push --tags
-
-Push elektra-ftp:
- cd ~elektra-ftp && git push && git push github
-
-Push elektra-doc to github:
- cd ~elektra-doc && git push && git push github
+== Preperation for next release == 
 
+- Increment CMPVERSION in scripts/build/run_icheck 
+- Cleanup tests/icheck.suppression (and add info to release notes)
+- Increment version in output of doc/tutorials/hello-elektra.md
+- Increment <Version> of libelektra4j in examples/external/java/read-keys-example/pom.xml and src/bindings/jna/README.md
```

Comment:
Can't we do this together with the other version increments?

Reply:
I was wondering too if this can be done earlier. Actually it probably _should_ happen earlier otherwise the release builds a wrong/old version of `libelektra4j`.